src/Element/DropzoneJs.php

```diff
@@ -76,6 +79,15 @@ public static function processDropzoneJs(&$element, FormStateInterface $form_sta
       '#attributes' => ['data-upload-path' => \Drupal::url('dropzonejs.upload')],
     ];
 
+    if (empty($element['#multiple'])) {
```

Comment:
This never gets set AFAIK and I created www.drupal.org/node/2723287 to clean this up.